src/content/en/2022/javascript.md

```diff
@@ -16,7 +17,1010 @@ featured_stat_2: TODO
+        <td class="numeric">0.93%</td>
+      </tr>
+      <tr>
+        <td>Modernizr, jQuery</td>
+        <td class="numeric">0.87%</td>
+        <td class="numeric">0.86%</td>
+      </tr>
+      <tr>
+        <td>core-js</td>
+        <td class="numeric">0.92%</td>
+        <td class="numeric">0.85%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(
+    caption="Analysis of libraries and frameworks used together on observed pages.",
+    sheets_gid="1200090523",
+    sql_file="frameworks_libraries_combos.sql"
+  ) }}</figcaption>
+</figure>
+
+It's clear though that jQuery has some serious staying power, with some combination of it, its UI framework, and its migration plugin occurring in the top seven spots, with core-js having a prominent role in library usage as well.
+
+#### Security vulnerabilities
+
+Given the wide proliferation of JavaScript on today's web, and with the advent of installable JavaScript packages, it's no surprise that security vulnerabilities exist in the JavaScript ecosystem.
+
+{{ figure_markup(
+    caption="The percentage of mobile pages that download a vulnerable JavaScript library or framework.",
+    content="57%",
+    classes="big-number",
+    sheets_gid="1881889053",
+    sql_file="lighthouse_vulnerabilities.sql"
+  )
+}}
+
+While 57% of mobile pages serving up a vulnerable JavaScript library or framework is significant, this figure is down from last year's figure of 64%. This is encouraging, but there's quite a bit of work to be done to lower this figure. We hope that as more security vulnerabilities are patched, developers will be incentivized to update their dependencies to avoid exposing their users to harm.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Library or framework</th>
+        <th>Desktop</th>
+        <th>Mobile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>jQuery</td>
+        <td class="numeric">49.12%</td>
+        <td class="numeric">48.80%</td>
+      </tr>
+      <tr>
+        <td>jQuery UI</td>
+        <td class="numeric">16.01%</td>
+        <td class="numeric">14.88%</td>
+      </tr>
+      <tr>
+        <td>Bootstrap</td>
+        <td class="numeric">11.53%</td>
+        <td class="numeric">11.19%</td>
+      </tr>
+      <tr>
+        <td>Moment.js</td>
+        <td class="numeric">4.54%</td>
+        <td class="numeric">3.91%</td>
+      </tr>
+      <tr>
+        <td>Underscore</td>
+        <td class="numeric">3.41%</td>
+        <td class="numeric">3.11%</td>
+      </tr>
+      <tr>
+        <td>Lo-Dash</td>
+        <td class="numeric">2.52%</td>
+        <td class="numeric">2.44%</td>
+      </tr>
+      <tr>
+        <td>GreenSock JS</td>
+        <td class="numeric">1.65%</td>
+        <td class="numeric">1.62%</td>
+      </tr>
+      <tr>
+        <td>Handlebars</td>
+        <td class="numeric">1.27%</td>
+        <td class="numeric">1.12%</td>
+      </tr>
+      <tr>
+        <td>AngularJS</td>
```

Comment:
Can we link to the [AngularJS](https://angularjs.org)' website? We did poor job with branding and people still think that's [Angular](https://angular.io).